matcher/encoder.py

```diff
@@ -87,20 +98,11 @@ def _encode_scores(self, scores, default=0):
         score_matrix = np.full(self.matrix_shape, default, dtype=float)
 
         for forum, user, score in scores:
-            if not isinstance(score, float) and not isinstance(score, int):
-                try:
-                    score = float(score)
-                except ValueError:
-                    raise EncoderError(
-                        'could not convert score {} of type {} to float ({}, {})'.format(
-                            score, type(score), forum, user))
-
-            # sometimes papers or reviewers get deleted after edges are created,
-            # so we need to check that the head/tail are still valid
-            if forum in self.papers and user in self.reviewers:
+            if user in self.index_by_user:
```

Comment:
don't you need to remove this if too, it seems the interface is also checking this